gcl_sdk/audit/api/controllers.py

```diff
@@ -0,0 +1,28 @@
+import uuid as sys_uuid
+
+from gcl_iam import controllers as iam_controllers
+from restalchemy.api import controllers, resources
+
+from gcl_sdk.audit.dm import models
+
+
+class AuditController(
+    iam_controllers.PolicyBasedControllerMixin,
+    controllers.BaseResourceControllerPaginated,
+):
+    __policy_service_name__ = "audit_log"
+    __policy_name__ = "audit_record"
+    __resource__ = resources.ResourceByRAModel(
+        model_class=models.AuditRecord,
+        process_filters=True,
+        convert_underscore=False,
+    )
+
+    def filter(self, filters, order_by=None):
+        self._enforce("read")
```

Comment:
This check doesn't restrict data by project scope. Why don't just use PolicyBasedController as parent? It's filter and get methods already have everything needed. Did I miss something?